src/Transport/Http.php

```diff
@@ -222,4 +222,22 @@ protected function _getConnection(bool $persistent = true)
 
         return self::$_curlConnection;
     }
+
+    protected function _getAuthType()
+    {
+        switch ($this->_connection->getAuthType()) {
+            case "digest":
+                return CURLAUTH_DIGEST;
+                break;
+            case "gssnegotiate":
+                return CURLAUTH_GSSNEGOTIATE;
+                break;
+            case "ntlm":
+                return CURLAUTH_NTLM;
+                break;
+            case "basic":
+            default:
+                return CURLAUTH_BASIC;
```

Comment:
Before the default value was `CURLAUTH_ANY`. I wonder if this change could break some setups?

Reply:
I changed the code and now the _getAuthType function returns CURLAUTH_ANY by default.

Reply:
I added changelogs